inspirehep/modules/records/jsonschemas/records/authors.json

```diff
@@ -247,9 +247,27 @@
                     "type": "string"
                 },
                 "properties": {
-                    "name": {
-                        "type": "string"
-                    }
+                  "name": {
+                    "type": "string"
+                  }
+                },
+                "name_variations": {
```

Comment:
This should not be part of the schema afaik, as it is something automatically computed that is only in ES. Not something that, for example, would be edited by a cataloger.